config/bili.config.js

```diff
@@ -1,27 +1,23 @@
 const path = require('path');
 
-// TODO: try `sourcemap` & `open` options once we migrate to latest version
-//       of rollup-plugin-visualizer which is currently locked due to older
-//       rollup version used by bili
 const visualizer = {
-  // sourcemap: true,
-  // open: true
+  // open: true, - the build process hangs if option is set
+  sourcemap: true
 };
 
 module.exports = {
-  name: 'tailor-teaching-elements',
-  moduleName: 'TailorTeachingElements',
-  format: ['umd', 'umd-min', 'es'],
-  plugins: ['vue', 'node-builtins', 'visualizer'],
-  postcss: {
-    extract: false
+  babel: { babelrc: false },
+  plugins: {
+    vue: true,
+    'node-builtins': true,
+    alias: { '@': path.resolve(__dirname, '../src') },
+    visualizer
   },
-  babel: {
-    babelrc: false,
-    extensions: [ 'js', '.vue' ]
-  },
-  visualizer,
-  alias: {
-    '@': path.resolve(__dirname, '../src')
+  output: {
+    extractCSS: false,
+    format: ['esm', 'esm-min', 'umd', 'umd-min'],
+    fileName: 'tailor-teaching-elements[min].[format][ext]',
```

Comment:
What's the default name without it? 🤔

Reply:
They start with `index` instead of `tailor-teaching-elements`. I assume it takes the name from the input file but I have not looked into it.

Reply:
Which is why @zcuric did this: https://github.com/zcuric/plyrue/blob/2f45cef/bili.config.js#L1-L4 💡